### ISSUE TITLE
Server crashing/lagging cheat

### ISSUE THREAD

**Issue by nikola924** [DDRace/teeworlds#189]
Hi,

someone made a new cheat that makes the server lag, like, a lot. Here's a demo of it: http://www.sendspace.com/file/j4ocyj

**Comment by AdVetter** [comment]
Demo shows fast reconnecting clients.

**Comment by nikola924** [comment]
Exactly. That causes the whole server to lag. If you press TAB while playing the demo, you'd see that everyone has ping on 1000.

**Comment by eeeee** [comment]
@nikola924 this is likely an old (https://www.teeworlds.com/forum/viewtopic.php?id=8947) issue in teeworlds and not only in ddrace, please refile it at https://github.com/teeworlds/teeworlds
